src/noc/containers/noc-check-and-submit/noc-check-and-submit.component.spec.ts

```diff
@@ -147,6 +147,27 @@ describe('NocCheckAndSubmitComponent', () => {
     expect(spyOnDispatchToStore).toHaveBeenCalledWith(new fromFeature.SetAffirmationError(affirmationError));
   });
 
+  it('should set notify error', () => {
```

Comment:
please change to:
"should set notify every party error when user doesn't notify every releted party"